app/panel/components/CreateAccount.jsx

```diff
@@ -200,6 +210,15 @@ class CreateAccount extends React.Component {
 									</div>
 								</div>
 							</div>
+							<div className="row">
+								<div className="small-12 columns">
+									<div id="create-account-legal-consent-checkbox" className={(legalConsentNotCheckedError ? 'checkbox-error' : '')}>
+										<input id="legalConsentChecked" name="legalConsentChecked" type="checkbox" checked={legalConsentChecked} onChange={this.handleCheckboxChange} />
+										{/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
```

Comment:
Do we still need this lint exclusion?

Reply:
Without it, the jsx-a11y linter plugin complains that "A form label must be associated with a control". It thinks the label is empty because we are using dangerouslySetInnerHTML attribute to set the text instead of inserting the string directly into the tag as with the promotions checkbox. But if we don't use dangerouslySetInnerHTML, the links in the string are displayed as plain text HTML. It seemed to me that this was a rare case where disabling the linter is the lesser of two evils, the other evil being somehow changing the checkbox implementation and probably making it more complicated and inconsistent with the promotions checkbox implementation. What do you think?